aptos-move/e2e-move-tests/src/harness.rs

```diff
@@ -626,8 +632,7 @@ impl MoveHarness {
             ]);
     }
 
-    /// Increase maximal transaction size.
-    pub fn increase_transaction_size(&mut self) {
+    fn change_one_gas_param_from_default(&mut self, param: &str, param_value: u64) {
```

Comment:
nit: 
- maybe 'override' is a better name (if I understand the logic)
- since we do it this way, should we just provide a vector of params to override? (we can collect entries as HashMap, update overrides, and re-collect or smt)